src/jquery.i18n.js

```diff
@@ -260,16 +260,22 @@
 	$.i18n.parser = {
 		// The default parser only handles variable substitution
 		parse: function ( message, parameters ) {
-			return message.replace( /\$(\d+)/g, function ( str, match ) {
-				var index = parseInt( match, 10 ) - 1;
-				return parameters[ index ] !== undefined ? parameters[ index ] : '$' + match;
+			return message.replace( /(\\?\$\d+)/g, function ( str, match ) {
+				var index;
+
+				if ( match[ 0 ] === '\\' ) {
+					// Escaped.
+					return match.slice( 1 );
+				}
+				index = parseInt( match.slice( 1 ), 10 ) - 1;
+				return parameters[ index ] !== undefined ? parameters[ index ] : match;
 			} );
 		},
 		emitter: {}
 	};
 	$.i18n.fallbacks = {};
 	$.i18n.debug = false;
-	$.i18n.log = function ( /* arguments */ ) {
+	$.i18n.log = function ( /* arguments */) {
```

Comment:
jsbeautify does not play nicely with comment inside function argument- I just removed it.